libs/domain/product/src/mixins/product.mixin.ts

```diff
@@ -19,6 +20,7 @@ export declare class ProductMixinInterface
 {
   sku?: string;
   protected $product: Signal<Product | null>;
+  protected $productQualifier: Signal<ProductQualifier | null>;
```

Comment:
```suggestion

  /**
   * A signal representing the product qualifier.
   * If the feature version is 1.3 or higher, it's an object containing SKU and offer.
   * Prior to version 1.3, it's a string representing SKU only.
   */
  protected $productQualifier: Signal<ProductQualifier | null>;
```

Also, the interface seems not aligned (`undefined` vs `null`)

Reply:
actually it's aways qualifier, regardless the feature flag, as we are normalizing it inside, depending if context is a string (pre 1.3) or qualifier.